src/main/scala/TestRunner.scala

```diff
@@ -0,0 +1,34 @@
+import org.apache.spark.{SparkConf, SparkContext}
+import pipelines.Logging
+import tests.IntegrationTests._
+
+object TestRunner extends Logging {
+  def main(args: Array[String]) = {
+    val testName = if (args.length < 1) "all" else args(0)
+
+    val sc = new SparkContext(new SparkConf().setAppName("TestRunner: " + testName))
+
+    val tests = Map(
+      "cifar" -> cifar,
+      "voc" -> voc,
+      "newsgroups" -> newsgroups
+    )
+
+    val runs = testName match {
+      case "all" => tests.keys
+      case _ => Seq(testName)
+    }
+
+    runs.foreach(t => {
+      logInfo(s"Running test $t")
+      val start = System.currentTimeMillis()
+      tests(t)(sc)
+      val diff = System.currentTimeMillis() - start
+      logInfo(s"Finished test $t in $diff")
```

Comment:
I agree - there's a hacky way to do it and a clean way to do it. The hacky way to do it is to grep the logs for a magic string on a per-pipeline basis (VOC reports "MAP", CIFAR reports something else, etc.) 

The cleaner way to do it is to recreate these pipelines from scratch in this repo or change them in Keystone so that they expose these numbers programmatically (or at least return the fit pipeline when they're done). 

My thought for the first generation of this was to write a bash/python script to pull out the fields of interest from the log files generated by the TestRunner. (e.g. do the hacky thing) - what do you think?